virtualfish/environment.fish

```diff
@@ -0,0 +1,56 @@
+function __vfext_environment_activate --on-event virtualenv_did_activate
+    if test -f $VIRTUAL_ENV/virtualfish-environment
+        for line in (cat $VIRTUAL_ENV/virtualfish-environment)
```

Comment:
I was a little on the fence about this just because this version reads a little more clearly to me from a code comprehension standpoint.
```
for line in (foo)
    # Do work
end
```
However, I also like using built-ins and the prevailing language constructs. Based on what you said and what I read [here](https://github.com/fish-shell/fish-shell/issues/1147), the method you recommend is the way to go.